pkg/config/config.go

```diff
@@ -10,16 +10,19 @@ var (
 	MasterKeyPath      string
 	HudsonSecretPath   string
 	CredentialsXmlPath string
+	JsonOutput         bool
 )
 
 const (
 	empty           = ""
 	masterKeyDesc   = "(required) master.key file location"
 	secretDesc      = "(required) hudson.util.Secret file location"
 	credentialsDesc = "(required) credentials.xml file location"
+	jsonDesc        = "(optional) output json"
 	usage           = `Usage:
 
   jenkins-credentials-decryptor \
+    -j \
```

Comment:
I'd prefer a generic `-o json` instead of `-j`.  
This way I could do `-o text` etc.  
If the `-o` flag is not provided it should default to json as it looks much more readable than that block of text.